src/main/scala/mesosphere/marathon/storage/StorageConfig.scala

```diff
@@ -252,11 +221,4 @@ object StorageConfig {
       case CuratorZk.StoreName => CuratorZk(conf, lifecycleState)
     }
   }
-
-  def apply(conf: Config, lifecycleState: LifecycleState): StorageConfig = {
```

Comment:
I thought @zen-dog removed this already.

Reply:
I did!